Extension/src/LanguageServer/extension.ts

```diff
@@ -1379,3 +1381,66 @@ export async function getIncludes(maxDepth: number): Promise<any> {
     const includes = await clients.ActiveClient.getIncludes(maxDepth);
     return includes;
 }
+
+async function onOTFDocs(): Promise<void> {
+    const response = await clients.ActiveClient.getOTFDocsInfo();
```

Comment:
Nit: `response` is a little confusing here given that we have a `chatResponse` further down in the function. Maybe something like `otfDocsInfo` instead?